web/source/keymanweb.ts

```diff
@@ -2846,7 +2869,18 @@ if(!window['keyman']['initialized']) {
      *                LisVirtualKey     e.g. Virtual key or non-keypress event
      */    
     keymanweb._GetKeyEventProperties = function(e, keyState) {
-      var s = new Object();
+      class KeyEvent {
+        Ltarg: any;
```

Comment:
I guess these types will be tightened later?

Reply:
The original goal for this PR was to perform as minimal actual TS conversion as necessary, tightening things up with subsequent PRs.  There will be a lot of work to knit things together better over time; while the `KeyEvent` class type is defined here, `keymanweb._GetKeyEventProperties` is not properly defined and TS cannot track its type specification with the method's declaration style.